[2021-10-02 04:10:37,991] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_A 2021-10-02T03:54:22.719241+00:00 [queued]>
[2021-10-02 04:10:38,125] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_A 2021-10-02T03:54:22.719241+00:00 [queued]>
[2021-10-02 04:10:38,128] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-10-02 04:10:38,131] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-10-02 04:10:38,134] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-10-02 04:10:38,210] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): Training_Model_A> on 2021-10-02T03:54:22.719241+00:00
[2021-10-02 04:10:38,346] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'Training_Model_A', '2021-10-02T03:54:22.719241+00:00', '--job-id', '14', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpdfmwmi1s', '--error-file', '/tmp/tmp9ijy9fbu']
[2021-10-02 04:10:38,349] {standard_task_runner.py:77} INFO - Job 14: Subtask Training_Model_A
[2021-10-02 04:10:38,280] {standard_task_runner.py:52} INFO - Started process 17339 to run task
[2021-10-02 04:10:38,583] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.Training_Model_A 2021-10-02T03:54:22.719241+00:00 [running]> on host d6b4131456ad
[2021-10-02 04:10:38,908] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=Training_Model_A
AIRFLOW_CTX_EXECUTION_DATE=2021-10-02T03:54:22.719241+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-10-02T03:54:22.719241+00:00
[2021-10-02 04:10:38,924] {python.py:151} INFO - Done. Returned value was: 1
[2021-10-02 04:10:39,073] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=Training_Model_A, execution_date=20211002T035422, start_date=20211002T041037, end_date=20211002T041039
[2021-10-02 04:10:39,288] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-10-02 04:10:39,463] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
